docs/design/storagenode-graceful-exit/reports.md

```diff
@@ -0,0 +1,45 @@
+# Storage Node Graceful Exit - Reports
+
+[Graceful Exit Overview](overview.md)
+
+## Abstract
+
+A satellite operator needs to know that status of exiting nodes in order to process escrows. 
+
+## Background
+
+As a result of a Graceful Exit, satellite operators need to release the escrow to the exited storage node. This means we need a report for exited nodes. 
+
+This report should contain:
+- NodeID
+- Wallet address
+- The date the node joined the network
+- The date the node initiated the Graceful Exit
+- The date the node completed the exit
+- GB Transferred (amount of data the node transferred during exiting)
+
+A satellite operator needs a list of nodes that have initiated an exit, but have not completed. This means we need a report for exiting nodes.
+
+This report should contain:
+- NodeID
+- Wallet address
+- The date the node joined the network
+- The date the node initiated the Graceful Exit
+- GB Transferred (amount of data the node transferred during exiting)
+
+## Design
+
+Add satellite CLI command to list gracefully exited nodes between two dates. The report should query `nodes` table where `exit_finished_at >= start AND exit_finished_at <= end date`. 
```

Comment:
The query is wrong. It should be `exit_finished_at >= start AND exit_finished_at < end date`. The end date is the first of a month 0:00 and if a storage node hits that timestamp it would get a place in 2 reports.

Reply:
Updated